internal/engine/dockerprune/docker_pruner.go

```diff
@@ -60,6 +62,14 @@ func (dp *DockerPruner) OnChange(ctx context.Context, st store.RStore) {
 	curBuildCount := state.CompletedBuildCount
 	hasDockerBuild := state.HasDockerBuild()
 	nextToBuild := buildcontrol.NextManifestNameToBuild(state)
+	var imgSelectors []container.RefSelector
+	for _, m := range state.Manifests() {
+		for _, iTarg := range m.ImageTargets {
+			sel := iTarg.ConfigurationRef
```

Comment:
i'm pretty sure this should be DeploymentRef, not ConfigurationRef?

DeploymentRef is what it actually gets tagged with

Reply:
oh yep you're right.